src/libstd/io/stdio.rs

```diff
@@ -71,42 +71,32 @@ impl Read for StdinRaw {
         Initializer::nop()
     }
 }
+
 impl Write for StdoutRaw {
     fn write(&mut self, buf: &[u8]) -> io::Result<usize> { self.0.write(buf) }
     fn flush(&mut self) -> io::Result<()> { self.0.flush() }
 }
+
 impl Write for StderrRaw {
     fn write(&mut self, buf: &[u8]) -> io::Result<usize> { self.0.write(buf) }
     fn flush(&mut self) -> io::Result<()> { self.0.flush() }
 }
 
-enum Maybe<T> {
-    Real(T),
-    Fake,
-}
+struct Maybe<T> (T);
```

Comment:
That seems like a strange name for a type that always has a `T` ;)

Reply:
Yes :smile:. I am working on that part in a follow-up PR to handle when a terminal is connected to stdout, a pipe or nothing. But decided to split this part out as it can mostly stand on its own.

Reply:
Ha! 🤣 😂; that's fair enough ^.^

Reply:
But maybe leave a note for yourself in case you forget?